.github/workflows/codeql.yml

```diff
@@ -18,27 +18,21 @@ jobs:
     permissions:
       security-events: write
 
-    strategy:
-      fail-fast: false
-      matrix:
-        language:
-          - 'cpp'
-          - 'go'
-
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
```

Comment:
Would be good to update this one as well.

Reply:
Done!